### PR TITLE
Axes

### PR DESCRIPTION
**Pull request by kferris10**
@timelyportfolio Is this what you're thinking for #2?

If so, I'll make them look a little bit nicer then add an option to turn them off.

### CODE REVIEW

**Comment by timelyportfolio** [comment]
Yes, looks great.  Thanks.  I just ran out of time last week.  Are you planning to just use `CSS` for the styling or will you do in `JS`?

**Comment by kferris10** [comment]
I hadn't thought about it.  It would be nice to do something that would allow the user to easily tweak it.  Any thought/suggestions?

**Comment by kferris10** [comment]
@timelyportfolio I'll just add css styling for now.  If anybody makes a request, I can update it later.  I'm hoping to start using this at work so I may have more changes coming.

**Comment by kferris10** [comment]
Note: add axis labels.  Then this can be closed

File: inst/htmlwidgets/shotsign.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ HTMLWidgets.widget({
     // y = Field Goal % in shooting signatures
     var y = d3.scale.linear()
       .domain(xx.ydomain ? xx.ydomain : [0, 1])
-      .range([height, 0 + maxWidth]);
+      .range([height, 0 + margin.top]);
```
Comment (kferris10) [inline inst/htmlwidgets/shotsign.js:55]: @timelyportfolio I'm not entirely sure why this was set to be `maxWidth` in the original code, but it results in just having a lot of extra space at the top of most of my signatures which is kind of annoying.  Do you have any issues with this?

Comment (timelyportfolio) [inline inst/htmlwidgets/shotsign.js:55]: looks like a bug; feel free to change!  Thanks so much.

**Comment by timelyportfolio** [comment]
I really appreciate you pushing forward with this.  Thanks.

**Comment by kferris10** [comment]
You're welcome.  I feel like I should be thanking you though.  I really appreciate all the work that you've put into these widgets, and I've found this one in particular to be helpful.  

If you ever have a chance to take a look at #7 I'd love to know what you think.

**Pull request merged**